beacon-chain/state/testing/getters_checkpoint.go

```diff
@@ -0,0 +1,103 @@
+package testing
+
+import (
+	types "github.com/prysmaticlabs/eth2-types"
+	"github.com/prysmaticlabs/go-bitfield"
+	"github.com/prysmaticlabs/prysm/beacon-chain/state"
+	fieldparams "github.com/prysmaticlabs/prysm/config/fieldparams"
+	ethpb "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/testing/require"
+
+	"testing"
+)
+
+func VerifyBeaconStateJustificationBitsNil(t *testing.T, factory getState) {
+	s, err := factory()
```

Comment:
`factory` is a noun, which is not the best name for a function. Let's rename it to what the function returns e.g. `createFoo`.